src/App.js

```diff
@@ -1,29 +1,15 @@
-import React, { useEffect, useState } from 'react';
-import { getAllData } from './util/index';
+import React, { useState } from 'react';
 import UserRegistration from './components/UserRegistration';
 import SignIn from './components/SignIn';
 import Home from './components/Home';
 import { Routes, Route, Navigate } from 'react-router-dom';
 import Navbar from './components/Navbar';
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
 
-const URL = 'http://localhost:8000/api/v1/';
-
 function App() {
   const [message, setMessage] = useState('');
```

Comment:
Sorry, one more thing we can probably remove; Along wiht line 16 where `message` is used.

Reply:
Removed.